docs/grading.md

```diff
@@ -8,94 +8,121 @@ contributions and community involvement into appropriate grades. Generally
 speaking, if you're contributing to the community and/or making valuable
 open-source contributions you'll receive a high grade in RCOS.
 
-## Grade Levels
+## Project Contributions & Planning
 
-Beginning in Fall 2016, RCOS will begin accepting 4 credit levels for RCOS students.
-Higher credit levels generally entail more commitment and responsibility, while
-lower credit levels take less time and require less objectives.
+### Proposal
 
-The following table represents a general rule-of-thumb for metrics that every
-RCOS member should keep track of (see [grading metrics](#grading-metrics) for
-longer descriptions of each metric).
+Each project must start with a proposal. This proposal will serve as your plan and contract, and should describe a set of well
+defined features your team will implement over the course of the semester. There should
+be (4+?) features in the proposal and each should have a planned date of completion.
+Each team member should contribute to at least (2?) features, and multiple team members
```

Comment:
@mskmoorthy - I was thinking of perhaps assigning percentages to each requirement. This would give people a sense of how much effort they should put into each part.

Or, I also potentially like the idea of a table that qualitatively describes what is necessary for each grade level, e.g., "A: student makes consistent contributions following their proposal, and appropriately tracks their issues and reports their progress, makes a blog post for every feature they work on, has several forms of documentation, etc"

I'm not sure which approach students and mentors would like better.